.github/workflows/oppiabot.yml

```diff
@@ -4,13 +4,22 @@ on:
   issues:
     types:
       - labeled
+  pull_request_target:
+    branches:
+      - develop
+      - release-*
```

Comment:
Do we need this check for the release branch? I think we don't allow PR "not targeting to develop branch".

Reply:
No, we don't allow PR "not targeting to develop branch". Shall I remove the release then?

Reply:
> like when we have a problem cherry-picking a PR from develop to a release branch

@vojtechjelinek In that case, we directly resolve the conflicts and create new commits, right? (And push the changes directly to the release branch instead of creating a PR) [not sure!]

Reply:
We did that previously, but now we recommend creating a separate PR to the release branch so that the conflicts are fixed by the PR author and not by the release coordinator.